index.html

```diff
@@ -1,76 +1,200 @@
+<section class="aditional-actions">
   <div class="container">
-    <a class="news fixed" href="{{ site.posts.first.url }}"><strong>Check it out!</strong> {{ site.posts.first.title }}</a>
+    <a class="news fixed" href="{{ site.posts.first.url }}">
+      <strong>Check it out!</strong> {{ site.posts.first.title }}</a>
   </div>
 </section>
 
-     <section class="main-features">
-      <div class="container">
-
-
-        <h2 class="section-header">AeroGear Modules</h2>
-
-
-        <div class="row">
-
-          <div class="col-md-3 push">
-            <h3><a href="/push/"><i class="fa fa-paper-plane"></i> AeroGear<strong>Push</strong></a></h3>
-            <p class="alt">Send push notifications to any device, regardless of platform or network.</p>
-            <p>Send push notifications from the <strong>Unified Push Server</strong> via <span title="Google Cloud Messaging, Apple’s APNs, Amazon's ADM, Microsoft's WNS and MPNS, SimplePush, WebPush">7 push networks</span> to any mobile device.</p>
-
-            <p><a href="/push/" class="btn btn-primary">Learn about AeroGear<strong>Push</strong></a></p>
-
-          </div><!-- col -->
-
-          <div class="col-md-3 security">
-            <h3><a href="/security/"><i class="fa fa-shield"></i> AeroGear<strong>Security</strong></a></h3>
-            <p class="alt">Integrate with your existing security and authentication infrastructure.</p>
-            <p>A small set of classes to provide integration with security providers written in Java, like <strong>PicketLink</strong> and <strong>Apache Shiro</strong>.</p>
-            <p><a href="/security/" class="btn btn-primary">Learn about AeroGear<strong>Security</strong></a></p>
-          </div><!-- col -->
-
-
-          <div class="col-md-3 sync">
-            <h3><a href="/sync/"><i class="fa fa-refresh"></i> AeroGear<strong>Sync</strong></a></h3>
-            <p class="alt">Real-time, client-server data synchronization with ease and efficiency.</p>
-            <p>Full real-time data sync where updates are initiated from both the client and server over a bi-directional channel.</p>
-            <p><a href="/sync/" class="btn btn-primary">Learn about AeroGear<strong>Sync</strong></a></p>
-          </div><!-- col -->
-
-          <div class="col-md-3 sync">
-              <h3><a href="/digger/"><i><img src="img/aerogeardigger_icon_32px_cropped.png"></i> AeroGear<strong>Digger</strong></a></h3>
-              <p class="alt">Build your mobile apps in the cloud with pipelines.</p>
-              <p>Build mobile apps in an isolated, scalable and deterministic way in the cloud.</p>
-              <p><a href="/digger/" class="btn btn-primary">Learn about AeroGear<strong>Digger</strong></a></p>
-          </div><!-- col -->
-
-
-        </div><!-- row -->
-      </div><!-- container -->
-    </section><!--main features-->
-
-
+<section class="main-features">
+  <div class="container">
+    <h2 class="section-header">AeroGear Services</h2>
```

Comment:
small note here on the heading. not sure it works in the black at the side. Suggestion to centre it and change to either the blue or yellow color. Not sure what's the right solution so @jasonmadigan @dimitraz feel free to add your thoughts here.

Reply:
Agree on centering it :+1: . I'm just not sure chaning it to yellow as it is against a white background. With blue, it may not stand out as all the other headings and links are in blue.

Reply:
@JameelB yes very good points, i'm not sure what the solution is.

Reply:
Agreed centering will look a lot better :+1: